pkg/controller/nmstate/nmstate_controller_test.go

```diff
@@ -16,32 +22,86 @@ import (
 	"github.com/nmstate/kubernetes-nmstate/pkg/names"
 )
 
+func copyManifest() {
```

Comment:
We have to mimic
```
COPY deploy/crds/nmstate.io_nodenetwork*.yaml /bindata/kubernetes-nmstate/crds/
COPY deploy/handler/namespace.yaml /bindata/kubernetes-nmstate/namespace/
COPY deploy/handler/operator.yaml /bindata/kubernetes-nmstate/handler/handler.yaml
COPY deploy/handler/service_account.yaml /bindata/kubernetes-nmstate/rbac/
COPY deploy/handler/role.yaml /bindata/kubernetes-nmstate/rbac/
COPY deploy/handler/role_binding.yaml /bindata/kubernetes-nmstate/rbac/
```
So both CRDs can be use a wildcard

Also can we do a `copyFile` wrapper function to reduce code here ? so we do 
`copyFile("..", "...") 

Also check errors here with `Expect(err).ToNot(HaveOcurred())`

Reply:
I think that I addressed most of your comments here